ADR/0026-appstudio-object-labels.md

```diff
@@ -0,0 +1,36 @@
+# 26. Record architecture decisions
+
+Date: 2023-06-02
+
+## Status
+
+Accepted
+
+## Context
+
+There is a growing number of CRDs and other objects that together describe the application and its constituent parts. We need to use a consistent set of labels to describe to which
+part of the application certain supporting objects belong. This is not a functional requirement as such but a consistent set of labels supports troubleshooting efforts as well
+as general hygiene in the cluster (by being able to answer the questions like "what are all the objects associated with this environment?").
+
+Consistent set of labels will also enable finding related objects when co-operation between several components is required yet directly referencing the related objects in the specs or statuses of
+other objects is not feasible or practical.
+
+## Decision
+
+The deployments, pods, configmaps, secrets, etc. that together compose AppStudio itself will have the usual common suggested set of labels such as:
+* `app.kubernetes.io/name = has`
+* `app.kubernetes.io/part-of = appstudio`
+
+The more important part, though, is to identify the supporting objects defining the application model within AppStudio. This proposal establishes the 3 basic labels that can identify objects that are related to the most important parts of the model - application, environment and component:
```

Comment:
Is there any criteria that we should have if we want to expand from this core set of three basic labels?